libraries/load-themed-styles/src/index.ts

```diff
@@ -164,6 +164,9 @@ function initializeThemeState(): IThemeState {
  * @param {boolean} loadAsync When true, always load styles in async mode, irrespective of current sync mode.
  */
 export function loadStyles(styles: string | ThemableArray, loadAsync: boolean = false): void {
+  if (typeof document === 'undefined') {
```

Comment:
>if (typeof document === 'undefined') { [](start = 2, length = 38)

Could you add a code comment indicating the usage scenario where "document" might be undefined?

Your GitHub issue discussed this, but someone reading this code wouldn't have that context

Reply:
Shouldn't it be trivial for anyone looking at this to know this just exits on non-DOM environments? Also this escape hatch is used further down without any comments.

Reply:
Fair enough